charts/podinfo/values-prod.yaml

```diff
@@ -107,6 +107,9 @@ serviceAccount:
 # set container security context
 securityContext: {}
 
+# set pod security context
+podSecurityContext: {}
```

Comment:
Can you please add this to the chart readme in the values table? Thanks

Reply:
Ah yes, I did not see it wasn't automated. Done!